examples/delay_actor.rs

```diff
@@ -0,0 +1,145 @@
+use anyhow::Error;
+use env_logger::Env;
+use log::warn;
+use std::{
+    cmp::Ordering,
+    collections::BinaryHeap,
+    time::{Duration, Instant},
+};
+use tonari_actor::{Actor, Context, Recipient, SendError, System};
+
+/// Something that can be sent. A message with a recipient.
+trait Envelope {
+    fn send(self: Box<Self>) -> Result<(), SendError>;
```

Comment:
Trivia: it was tricky for me to figure out the need to wrap `Self` in a `Box` here. But without it, calling `message.send()` given `message: Box<dyn Envelope + Send>` didn't work: `rustc` complained it cannot move a type whose size is not known at compile time. Wrapping in a Box moves the move (heh) into the trait implementation, where it is known. [Method call operator still works](https://doc.rust-lang.org/reference/items/associated-items.html#methods).